service-python/assessclaimdc6510/src/lib/queues.py

```diff
@@ -13,20 +13,20 @@
 def on_request_callback(channel, method, properties, body):
     binding_key = method.routing_key
     message = json.loads(body.decode("utf-8"))
-    logging.info(f" [x] {binding_key}: Received message.")
+    logging.info(f"claimSubmissionId: {message['claimSubmissionId']}, health data received by {binding_key} processor")
```

Comment:
Very nice! I had not realized that Java added python-style formatting. Now I am going to use this everywhere :)